python/peacock/ExodusViewer/plugins/VTKWindowPlugin.py

```diff
@@ -393,16 +361,9 @@ def _renderResult(self):
 
         # Create result (renderer)
         self._result = chigger.exodus.ExodusResult(self._reader, **self._result_options)
+        self._window.append(self._result)
```

Comment:
@aeslaughter Moving this line back to where it was (actually after `self.setupResult.emit(self._result)`), makes the 3d viewing great again